app/src/test/java/com/codesoom/assignment/service/CatToyServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.domain.CatToy;
+import com.codesoom.assignment.repository.CatToyRepository;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+
+@Nested
+@DisplayName("CatToyService 클래스")
+class CatToyServiceTest {
+
+    private CatToyServiceImpl catToyServiceImpl;
+    private CatToyRepository catToyRepository;
+
+    private CatToy givenCatToy;
+    private List<CatToy> givenCatToyList;
+
+    private static final Long EXIST_ID = 1L;
+
+    @BeforeEach
+    public void setUp() {
+        catToyRepository = mock(CatToyRepository.class);
+        catToyServiceImpl = new CatToyServiceImpl(catToyRepository);
+
+        givenCatToy = new CatToy(
+                EXIST_ID,
+                "Test Name",
+                "Test Maker",
+                10000,
+                "test Image Url"
+        );
+
+        givenCatToyList = new ArrayList<>();
+        givenCatToyList.add(givenCatToy);
+    }
+
+    @Nested
+    @DisplayName("getCatToys 메서드는")
+    class getAllCatToys {
+        @BeforeEach
+        void prepareTest() {
+            given(catToyRepository.findAll()).willReturn(givenCatToyList);
+        }
+
+        @Test
+        @DisplayName("장난감 목록 전체를 반환한다.")
+        void getCatToys() {
+            List<CatToy> catToys = catToyServiceImpl.getCatToys();
+            verify(catToyRepository).findAll();
+            Assertions.assertThat(catToys).hasSize(1);
```

Comment:
```suggestion
            List<CatToy> catToys = catToyServiceImpl.getCatToys();

            verify(catToyRepository).findAll();

            Assertions.assertThat(catToys).hasSize(1);
```

테스트 코드와 단정문은 다른 일을 하고 있으니, 적절히 빈 줄로 구분해주면 좋을 것 같아요.

Reply:
넵 의미 단위로 띄어주는 것이 더 좋은 것 같습니다!